pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: material2
+environment:
+  sdk: '>=1.12.0-dev.5.10 <2.0.0'
```

Comment:
Not for right now, but we need to make sure this stays in sync with `angular/angular`